libraries/support/azure/service.rb

```diff
@@ -52,12 +66,16 @@ def set_reader(name, value, override)
       self
     end
 
-    def get(url:, api_version:)
+    def get(url:, api_version:, error: nil, unwrap: nil)
```

Comment:
I wonder about this name, `error`. It's really an optional error handler, right?